src/batched/ode/tftk_DeviceODESolvers.h

```diff
@@ -0,0 +1,192 @@
+/*--------------------------------------------------------------------*/
+/*    Copyright 2002 - 2008, 2010, 2011 National Technology &         */
+/*    Engineering Solutions of Sandia, LLC (NTESS). Under the terms   */
+/*    of Contract DE-NA0003525 with NTESS, there is a                 */
+/*    non-exclusive license for use of this work by or on behalf      */
+/*    of the U.S. Government.  Export of this program may require     */
+/*    a license from the United States Government.                    */
+/*--------------------------------------------------------------------*/
+#ifndef SIERRA_tftk_DeviceODESolvers_h
```

Comment:
This should be the opposite fpr dependency. KK cannot check for SIERRA, SIERRA will check for KK and pick the other option if KK is not there

Reply:
Yes.  Just hadn't finished removing all the sierra stuff yet.